finders/find_name.py

```diff
@@ -1,7 +1,30 @@
 from typing import List
-
 from finders.stops import Stop
+import re
 
 
 def find_stops_matching_name(stops: List[Stop], name: str, regex: bool = False) -> List[Stop]:
-    pass
+    found = False
+    results = list()
+    if not regex:
+        for element in stops:
+            if element.name == name:
+                print("Your stop is: ", element)
```

Comment:
Avoid letting debug print in your function, we'll have a look at the logging module next week